test/FactoriesSpec.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import components from './../tools/components';
+
+let props = {
+  Glyphicon: {glyph: 'star'},
+  Modal: {onRequestHide: function() {}},
+  ModalTrigger: {modal: React.DOM.div(null)},
+  OverlayTrigger: {overlay: React.DOM.div(null)}
+};
+
+function createTest(component) {
+  describe('factories', function () {
+    it(`Should have a ${component} factory`, function () {
+      let factory = require(`../lib/factories/${component}`);
```

Comment:
If you wrote files to `src/factories` than this could change to the src folder which would be in line with the rest of the unit tests. 

Also, you could have a [before hook](http://mochajs.org/#hooks) that writes missing factories to the `src/factories` folder. This is important for the [`npm run test-watch`](https://github.com/react-bootstrap/react-bootstrap/blob/v0.21.0/package.json#L12) which runs tests in watch mode without a build.